Mage.Sets/src/mage/cards/f/FriendlyRivalry.java

```diff
@@ -0,0 +1,110 @@
+package mage.cards.f;
+
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.predicate.other.AnotherTargetPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.Target;
+import mage.target.common.TargetControlledCreaturePermanent;
+import mage.target.common.TargetCreaturePermanent;
+
+/**
+ *
+ * @author Susucr
+ */
+public final class FriendlyRivalry extends CardImpl {
+
+    private static final FilterControlledCreaturePermanent filter2 = new FilterControlledCreaturePermanent("other target legendary creature you control");
+
+    static {
+        filter2.add(new AnotherTargetPredicate(2));
+        filter2.add(SuperType.LEGENDARY.getPredicate());
+    }
+
+    public FriendlyRivalry(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{R}{G}");
+        
+
+        // Target creature you control and up to one other target legendary creature you control
+        // each deal damage equal to their power to target creature you don't control.
+        this.getSpellAbility().addEffect(new FriendlyRivalryEffect());
+
+        TargetControlledCreaturePermanent target1 = new TargetControlledCreaturePermanent();
+        this.getSpellAbility().addTarget(target1.setTargetTag(1));
+
+        TargetControlledCreaturePermanent target2 = new TargetControlledCreaturePermanent(0, 1, filter2, false);
+        this.getSpellAbility().addTarget(target2.setTargetTag(2));
+
+        TargetCreaturePermanent target3 = new TargetCreaturePermanent(1, 1, StaticFilters.FILTER_CREATURE_YOU_DONT_CONTROL);
```

Comment:
One target is the default so this can use the constructor that just takes a filter and no need to add another constructor

Reply:
cleaned that up.